db/structure.sql

```diff
@@ -1773,6 +1664,426 @@ CREATE MATERIALIZED VIEW public.reporting_facility_appointment_scheduled_days AS
+           FROM all_follow_ups_with_medical_histories
+          GROUP BY all_follow_ups_with_medical_histories.facility_id, (date(all_follow_ups_with_medical_histories.visited_at)), all_follow_ups_with_medical_histories.day_of_year
+        ), last_30_days AS (
+         SELECT (generate_series((CURRENT_TIMESTAMP - '30 days'::interval), CURRENT_TIMESTAMP, '1 day'::interval))::date AS date
+        )
+ SELECT rf.facility_region_slug,
+    rf.facility_id,
+    rf.facility_region_id,
+    rf.block_region_id,
+    rf.district_region_id,
+    rf.state_region_id,
+    last_30_days.date AS visit_date,
+    (date_part('doy'::text, timezone(( SELECT current_setting('TIMEZONE'::text) AS current_setting), timezone('UTC'::text, (last_30_days.date)::timestamp with time zone))))::integer AS day_of_year,
+    daily_registered_patients.daily_registrations_htn_all,
+    daily_registered_patients.daily_registrations_htn_male,
+    daily_registered_patients.daily_registrations_htn_female,
+    daily_registered_patients.daily_registrations_htn_transgender,
+    daily_registered_patients.daily_registrations_dm_all,
+    daily_registered_patients.daily_registrations_dm_male,
+    daily_registered_patients.daily_registrations_dm_female,
+    daily_registered_patients.daily_registrations_dm_transgender,
+    daily_registered_patients.daily_registrations_htn_and_dm_all,
+    daily_registered_patients.daily_registrations_htn_and_dm_male,
+    daily_registered_patients.daily_registrations_htn_and_dm_female,
+    daily_registered_patients.daily_registrations_htn_and_dm_transgender,
+    daily_follow_ups.daily_follow_ups_htn_all,
+    daily_follow_ups.daily_follow_ups_htn_female,
+    daily_follow_ups.daily_follow_ups_htn_male,
+    daily_follow_ups.daily_follow_ups_htn_transgender,
+    daily_follow_ups.daily_follow_ups_dm_all,
+    daily_follow_ups.daily_follow_ups_dm_female,
+    daily_follow_ups.daily_follow_ups_dm_male,
+    daily_follow_ups.daily_follow_ups_dm_transgender,
+    daily_follow_ups.daily_follow_ups_htn_and_dm_all,
+    daily_follow_ups.daily_follow_ups_htn_and_dm_male,
+    daily_follow_ups.daily_follow_ups_htn_and_dm_female,
+    daily_follow_ups.daily_follow_ups_htn_and_dm_transgender
+   FROM (((public.reporting_facilities rf
+     JOIN last_30_days ON (true))
+     LEFT JOIN daily_registered_patients ON (((daily_registered_patients.visit_date = last_30_days.date) AND (daily_registered_patients.facility_id = rf.facility_id))))
+     LEFT JOIN daily_follow_ups ON (((daily_follow_ups.visit_date = last_30_days.date) AND (daily_follow_ups.facility_id = rf.facility_id))))
+  WITH NO DATA;
+
+
+--
+-- Name: reporting_facility_state_daily_dimensions; Type: MATERIALIZED VIEW; Schema: public; Owner: -
+--
+
+CREATE MATERIALIZED VIEW public.reporting_facility_state_daily_dimensions AS
```

Comment:
Is this part of the other PR? Check this file once for the diff.

Reply:
okay